src/org/gk/database/DefaultInstanceEditHelper.java

```diff
@@ -66,20 +66,20 @@ public Long getDefaultPerson() {
     }
 	
 	/**
-	 * Get the default InstanceEdit. If null, null will be returned.
-	 * @return 
-	 * @see getDefaultInstanceEdit(Window)
+	 * Get the default InstanceEdit. 
+	 * @return Default InstanceEdit. If null, null will be returned.
```

Comment:
Might be more clear to say "Will return null if this has not yet been set".
(assuming that statement is actually true).

Reply:
Rephrased as `@return Default InstanceEdit (or null if none exists).`